ee-galleon-pack/src/main/resources/layers/standalone/batch-jberet/layer-spec.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" ?>
+<layer-spec xmlns="urn:jboss:galleon:layer-spec:1.0" name="batch-jberet">
+    <dependencies>
+        <!-- CDI should be optional but to be conservative in the first cut at this, require it
+        <layer name="cdi" optional="true"/>
+        -->
```

Comment:
I missed this on the other PR, but I don't think CDI should be optional. Batch requires some kind of DI. I guess though if the user provides one that does make it optional.